packages/file-store/package.json

```diff
@@ -21,12 +21,10 @@
     "test": "mocha test.ts --exit --extension ts --require ts-node/register"
   },
   "dependencies": {
-    "configstore": "^5.0.1",
+    "@tus/server": "workspace:*",
     "debug": "^4.3.3"
   },
   "devDependencies": {
-    "@tus/server": "workspace:^",
```

Comment:
I see that this notation `workspace:^` is often used. But in documentation I only find `workspace:*`. Do you know what is a difference?

Reply:
Here is an example of input and output:

```js
{
  "dependencies": {
    "star": "workspace:*",
    "caret": "workspace:^",
    "tilde": "workspace:~",
    "range": "workspace:^1.2.3",
    "path": "workspace:path/to/baz"
  }
}
```

```js
{
  "dependencies": {
    "star": "1.5.0",
    "caret": "^1.5.0",
    "tilde": "~1.5.0",
    "range": "^1.2.3",
    "path": "1.5.0"
  }
}
```